pkg/operator/workloads/api_workload.go

```diff
@@ -234,10 +234,11 @@ func (aw *APIWorkload) IsFailed(ctx *context.Context) (bool, error) {
 }
 
 type downloadContainerArg struct {
-	From     string `json:"from"`
-	To       string `json:"to"`
-	Unzip    bool   `json:"unzip"`
-	ItemName string `json:"item_name"` // name of the item being downloaded, just for logging (if "" nothing will be logged)
+	From         string `json:"from"`
+	To           string `json:"to"`
+	Unzip        bool   `json:"unzip"`
+	ItemName     string `json:"item_name"`     // name of the item being downloaded, just for logging (if "" nothing will be logged)
+	SingleRename string `json:"single_rename"` // e.g. if path/to/target is set, path/to/* will be renamed to path/to/target only if there is one item in path/to/
```

Comment:
SingleRename is kind of confusing. What if we did it where if `from` ends with `/` or `/*` then you copy the contents of the folder to the folder specified in To?